### ISSUE TITLE
MonoDevelop FSharp.Core.dll not found

### ISSUE THREAD

**Issue by fellz** [fsharp/fsharp#571]
Hi
I've installed MonoDevelop at Ububtu 14.04. Steps i took:

Installed this

```
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF

echo "deb http://download.mono-project.com/repo/debian wheezy main" | sudo tee /etc/apt/sources.list.d/mono-xamarin.list

sudo apt-get update
 ```

Installed this

```
sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/tpokorra:/mono/xUbuntu_14.04/ /' >> /etc/apt/sources.list.d/monodevelop-opt.list"

sudo apt-get update

sudo apt-get install monodevelop-opt
```

Then ```monodevelop-opt``` runs MonoDevelop ver. 5-10

Also installed ```sudo apt-get install mono-complete fsharp```

I found Documentation pretty confusing. 
For example, in order to install "MonoDevelop" you need to install Mono project - ok. Then 

> After you installed Mono and added the package repository, install the monodevelop package to get the main MonoDevelop application. 

and no link to **monodelop** or instructions how i can install it. Finally i found a way to install  "MonoDevelop" IDE with ```monodevelop-opt```

So bug is pretty old here and according to github issues it was resolved couple of times in 2012 and 2013 and 2014. 2016 and it still here. 

How i can fix that bug and what is needed to close this bug forever ? 

p.s. Fix documentation plz in "MonoDevelop" project.

![fsharp-bug](https://cloud.githubusercontent.com/assets/96966/14879204/ee74b230-0d38-11e6-96f3-77695c61a6f3.jpg)

**Comment by 7sharp9** [comment]
This is the F# language repo, it's not related to F# in MonoDevelop, bugzilla is used for issues: https://bugzilla.xamarin.com/enter_bug.cgi?product=Xamarin%20Studio

**Comment by fellz** [comment]
I know but newbies like me will go to this repo i guess also github issues system simpler and more familiar to devs. So is there simple solution to this problem or i have to ask xamarin devs via bugzilla ?

**Comment by fellz** [comment]
Ok i found a solution. Changed Mono version in Preferences. May be it will help somebody

![mono-runtime](https://cloud.githubusercontent.com/assets/96966/14882767/6c09a130-0d4b-11e6-9070-4fe612ba67f5.jpg)

![selection_241](https://cloud.githubusercontent.com/assets/96966/14882778/79fbe0be-0d4b-11e6-8454-191403e150ce.jpg)

**Comment by 7sharp9** [comment]
Ok, glad you resolved it. A similar issue was fixed recently in Xamarin Studio 6.1 relating to F# core .Net 4.6+ references.

**Comment by dsyme** [comment]
Closing as external
